README.rst

```diff
@@ -0,0 +1,105 @@
+image here
+image target here
+
+image here
+image target here
+
+
+PDFmorph
+========================================================================
+add description, sample below
+Code package for morphin' atomic pair distribution functions.
+
+PDFmorph is a user-friendly software package. [more info here]
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3.7 and several third party libraries 
```

Comment:
is 3.7 a requirement or any p3?

Reply:
I think I spoke with @CJ-Wright and we talked about 3.7 being the minimum? But I might be misremembering! (sorry by the way, I just saw these comments now)

Reply:
please resolve.